apps/admin/src/app/workspace/tabs/audit-logs/workspace-audit-logs-tab.component.scss

```
@@ -0,0 +1,274 @@
+.json-grid h4 {
+  margin: 0 0 0.375rem;
+}
+
+pre {
+  min-height: 8rem;
+  max-height: 20rem;
+  margin: 0;
+  overflow: auto;
+  padding: 0.75rem;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)